nautobot/core/settings.py

```diff
@@ -567,6 +577,10 @@
         90,
         "Number of days to retain object changelog history.\nSet this to 0 to retain changes indefinitely.",
     ],
+    "DEPLOYMENT_ID": [
+        "",
+        "Randomly generated UUID used to identify this installation.\nUsed for sending anonymous installation metrics.",
```

Comment:
Do we care about enforcing this to be a UUID, or are arbitrary strings fine?

Also, maybe mention INSTALLATION_METRICS_ENABLED here?

Reply:
I spoke with @chadell about this: In the backend, it's implemented as a CharField so any string should be fine and there are no plans to change this.